memgpt/cli/cli.py

```diff
@@ -584,6 +585,15 @@ def run(
             )
             agent_state.llm_config.model_endpoint_type = model_endpoint_type
 
+        # user specified a new system prompt
```

Comment:
@sarahwooders I modified the way it works to be like this:

`memgpt run --system "my prompt"`

- if you create a new agent, the system prompt is initialized with this
- if you load an existing agent, the system prompt is overwritten
  - we override `agent.agent_state.state["system"]`
  - NOT `agent.agent_state.system` (which AFAIK is the "original" system prompt)

Reply:
however, I wonder if we should even include the ability to override via the CLI at all, since it's pretty dangerous + the user can use `/systemswap` after loading the agent (which gives the user a warning indicator - "are you sure").

if we don't want this, we can just comment out / delete this code block